rllib/execution/train_ops.py

```diff
@@ -511,21 +511,10 @@ def __call__(
 class UpdateTargetNetwork:
```

Comment:
writing the comment here, but it's not related to UpdateTargetNetwork.
can we delete the TrainOneStep and MultiGPUTrainOneStep classes now??